contrib/node/examples/src/node/hello-test/npm-shrinkwrap.json

```diff
@@ -0,0 +1,1983 @@
+{
```

Comment:
The only dependency I added is the compiler for ES2017, which is a very basic dependency required by most of the node package. It is pretty normal for node package to have this amount of dependencies.